src/getUserEmoji.js

```diff
@@ -169,11 +137,11 @@ function getUserIdNameMap(userInfo) {
 }
 
 async function replaceNamesEmojis(token, userIdEmojiList) {
-  let userNameEmojiMaps = []
-  let formattedMojis = []
-  let favoriteEmojis = []
-  const allUsers = await allUsersLookup(usersUrl, token)
-  const users = await getUserIdNameMap(allUsers)
+  const userNameEmojiMaps = []
```

Comment:
JavaScript's `const` keyword doesn't do much: it just prevents reassigning the variable. Things with 'interior mutability' like objects and arrays are still mungible when they're const.

Reply:
Ah! Yeah, for a while I thought I had to use `let` when declaring an empty array so that I'd be able to push entries to it later 😄 . The distinction between objects with 'interior mutability' and primitives without is v helpful.